src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -505,6 +510,11 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
       this.currentPage++;
       this.numberOfValues = this.options.numberOfValues + this.currentPage * this.options.pageSize;
       this.reload();
+      this.usageAnalytics.logCustomEvent<IAnalyticsFacetMeta>(
```

Comment:
You could create a private method for logging and use it for show more and show less (only passing the cause as parameter)

Reply:
Yeah, you're right I'll do that for the 3 facets. it's cleaner,